### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Your Subscriptions_Screen Reader: On T key press in Scan/Browse mode, screen reader focus is not shifting to the table

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4287]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.3.1; #ScreenReader; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; #Narrator; #NVDA 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator 

**MAS Violated:** MAS1.3.1

**NOTE:** Same issue is observed in the below screen
"Tools > Azure > Sign-in > Service Principle > New > Copy&Open > Create Authentication Files

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "Tools > Azure > Sign-n" in the navigation bar and press enter
4.Tab till "Device Login" radio button and select it 
5.Tab till "Sign-in" button and press enter
6.Tab till "Copy&Open" button and press enter 
7.Tab till "Enter Code" text box and paste the copied code. Tab till "Next" button and press enter
8.Tab till the required account and press enter
9.Go back to Eclipse IDE
10.Turn on Scan/Browse mode for screen reader
11.Verify if the screen reader focus is shifting to the subscriptions table on "T" key press

**Actual Result:**
On T key press in Scan/Browse mode, screen reader focus is not shifting to the subscriptions table

**Expected Result:**
On T key press in Scan/Browse mode, screen reader focus should shift to the subscriptions table

**User impact:**
Screen Reader users will find it difficult to navigate to the table if the shortcut keys are not working

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&action=default&mobileredirect=true&cid=8b75f338-0916-41c9-9fdf-ddb91b2b7698)

[Bug_4287_Java Tooling_Your Subscriptions_Table_Narrator.pptx](https://github.com/microsoft/azure-tools-for-java/files/4462316/Bug_4287_Java.Tooling_Your.Subscriptions_Table_Narrator.pptx)

![Bug_4287_Java Tooling_Service Principle_Your Subscriptions_Table_NVDA](https://user-images.githubusercontent.com/41108985/79004995-89b4bb80-7b73-11ea-9de9-70111109517a.png)
![Bug_4287_Java Tooling_Service Principle creation Status_Table_NVDA](https://user-images.githubusercontent.com/41108985/79004998-8b7e7f00-7b73-11ea-9d60-d40b4fd09650.png)
![Bug_4287_Java Tooling_Your Subscriptions_Table_NVDA](https://user-images.githubusercontent.com/41108985/79005002-8e796f80-7b73-11ea-84f1-8fd105a817d4.png)

**Comment by Flanker32** [comment]
@soumyatippapuram Seems NVDA didn't provide scan mode, this function is provided by narrator.
And this seems to be an issue of narrator scan mode or Eclipse, narrator can't shift to Tables in Eclipse, for instance, Windows -> Preferences -> Server -> Runtime Environment , this is the configuration panel of Eclipse, and it behaves the same as this issue, narrator will prompt no next table in scan mode

![image](https://user-images.githubusercontent.com/12445236/79313364-ce08d880-7f32-11ea-9963-b163691d6d95.png)

**Comment by Flanker32** [comment]
* Importance of experience: Low, users may get confused by narrator prompt "no next table" in scan mode, but they could tab to the table.
* Quality of experience: Users could tab to the table and get correct prompt message, so scan mode issue will not block users.
* Alternate solutions:  This issue will not break users experience, no need for alternate solutions
* Evidence: All evidence is provided in this issue.
* [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@soumyatippapuram This issue has been linked to AAT, please have a review, thanks!

**Comment by Alexandria-R** [comment]
Bug needs to be filed against Eclipse by product team. Once that is done then this bug can be closed as non blocking.

**Comment by Flanker32** [comment]
External issues tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=562549

**Comment by soumyatippapuram** [comment]
As external tracking bug is available, closing this issue

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749656
